modules/installation-configure-proxy.adoc

```diff
@@ -62,6 +65,19 @@ range that is specified in the `networking.machineNetwork[].cidr` field in the
 ====
 endif::bare-metal[]
 
+ifdef::kuryr[]
+[NOTE]
+====
+For Kuryr installations running on Restricted Network and using Proxy,
+the proxy setup has to allow proxy to reply into the OpenShift cluster's
+router.
+
+`sudo ip route add <CLUSTER_NETWORK_CIDR> via <INSTALLER_SUBNET_GATEWAY>`
```

Comment:
Should introduce the command explicitly, e.g. "To add a static route for the proxy configuration:"

Reply:
Good point, yes, better